### PR TITLE
Stg-master

### PR DESCRIPTION
**Pull request by kpslp**

### CODE REVIEW

File: support/troubleshooting-issues.md [hunk 1]
```diff
@@ -2,6 +2,168 @@
...
+3. Open *Terminal*.
+4. Type `sudo launchctl list | grep -i cloudflare` and ensure the service is running.
+5. If Cloudflare WARP is not installed or not running, refer to [troubleshooting Cloudflare issues](https://docs.developer.tech.gov.sg/docs/security-suite-for-engineering-endpoint-devices/support/troubleshooting-issues).
+
+> **Note**: Spotlight is used to detect the existence of applications. If you have disabled Spotlight, we would not be able to detect Cloudflare WARP on your device.
+
+**Ensure that Cloudflare is authenticated and verified**
+
+1. Click the *Cloudflare WARP* icon on the taskbar > *Preference*.
+2. Navigate to *Account* and ensure that you are authenticated with the team namenac *gccgovsg*.
```
Comment (cricketer94) [inline support/troubleshooting-issues.md:92]:
```suggestion
2. Navigate to *Account* and ensure that you are authenticated with the team name *gccgovsg*.
```

File: support/troubleshooting-issues.md [hunk 2]
```diff
@@ -2,6 +2,168 @@
...
+3. If Cloudflare WARP is not authenticated to *gccgovsg*, refer to [troubleshooting issues](https://docs.developer.tech.gov.sg/docs/security-suite-for-engineering-endpoint-devices/support/troubleshooting-issues).
+
+**For any other CloudFlare issues, refer to the [troubleshooting documentation](https://docs.developer.tech.gov.sg/docs/security-suite-for-engineering-endpoint-devices/support/troubleshooting-issues).**
+
+#### Intune
+
+**Ensure that Intune is enrolled with the valid tenant**
+
+1. Search for *Access work or school* in the search bar.
+2. Ensure that you have an account added with a *techpass.gov.sg* domain (for vendors) or a *.gov.sg* domain (for all Public officers).
```
Comment (cricketer94) [inline support/troubleshooting-issues.md:37]:
```suggestion
2. Ensure that you have an account added with a *techpass.gov.sg* domain (for vendors) or a *<agency_name>.gov.sg* domain (for all Public officers).
```

to make it consistent with mac

File: support/troubleshooting-issues.md [hunk 3]
```diff
@@ -2,6 +2,168 @@
...
+5. If it is still not enabled, raise a [service request](http://go.gov.sg/seed-techpass-support), and state *Disabled Real-Time protection and Cloud-delivered protection* in the description.
+
+**Ensure that Defender has a valid tenant**
+
+1. Search for *Run* in the search bar and type *regedit*.
+2. Navigate to *Computer\HKEY_LOCAL_MACHINE\SOFTWARE\Microsoft\Windows Advanced Threat Protection\Status* and look at the value of the *OrgID* key.
+3. Ensure that the value is one of the following:
+   - Public officers: faa36a5e-2da6-4225-8e27-226177c801a0
+   - Vendors: 49237d71-42ac-425a-a803-881b92cc18ce
+4. If the values are different, check with your IT administrator to identify which Defender tenant you are currently under and offboard from that tenant before attempting to re-onboard.
```
Comment (cricketer94) [inline support/troubleshooting-issues.md:72]:
```suggestion
1. Go to the Start menu and enter Powershell.
2. Right-click on the search result for PowerShell and select Run as Administrator
3. On Powershell, run the following command.
$reg64 = [Microsoft.Win32.RegistryKey]::OpenBaseKey([Microsoft.Win32.RegistryHive]::LocalMachine, [Microsoft.Win32.RegistryView]::Registry64)
$OrgID =  $reg64.OpenSubKey("SOFTWARE\MICROSOFT\Windows Advanced Threat Protection\Status").GetValue("OrgID")
echo $OrgID
4. Take note of the value displayed for OrgID.
5. Ensure that the value is one of the following:
   - Public officers: faa36a5e-2da6-4225-8e27-226177c801a0
   - Vendors: 49237d71-42ac-425a-a803-881b92cc18ce
6. If the values are different, check with your IT administrator to identify which Defender tenant you are currently under and offboard from that tenant before attempting to re-onboard.
```

Comment (cricketer94) [reply support/troubleshooting-issues.md:72]: Sorry ah huda, I copied the code block from here: https://docs.developer.tech.gov.sg/docs/security-suite-for-engineering-endpoint-devices/offboard-device/windows-offboarding-guide?id=phase-a-offboard-device-from-seed-components

Can help me set up the code block for here as well?

File: support/troubleshooting-issues.md [hunk 4]
```diff
@@ -2,6 +2,168 @@
...
+3. Ensure you see the following output.
+
+**If the service is not running, refer to the following steps**:
+
+1. Open *Terminal*.
+2. Run the following command to start the services:
+   - `sudo launchctl kickstart -k system/com.microsoft.wdav`
+   - `sudo launchctl kickstart -k system/com.microsoft.wdav.enterprise`
+3. Run `ps aux | grep -i "Defender"` to verify if the services are running now.
+4. If the issue persists, raise a [service request](http://go.gov.sg/seed-techpass-support).ma
```
Comment (cricketer94) [inline support/troubleshooting-issues.md:132]:
```suggestion
4. If the issue persists, raise a [service request](http://go.gov.sg/seed-techpass-support)
```

**Review by cricketer94 — CHANGES_REQUESTED** [review]
Hi Huda, some typos. 

I also want to add some images, but I think you can just address the typos first

File: support/troubleshooting-issues.md [hunk 5]
```diff
@@ -2,6 +2,168 @@
...
+
+**Ensure that Cloudflare service is installed and running**
+
+1. Search for *Add or Remove Programs* in the search bar.
+2. Ensure that Cloudflare Warp shows up as an installed program.
+![cf](/images/cf-installed.png)
+3. Search for *Run* in the search bar and run *services.msc*.
+4. Ensure that Cloudflare WARP is a running service.
+![cf run](/images/cf-running.png)
+5. If Cloudflare WARP is not installed or not running, refer to [troubleshooting Cloudflare issues](https://docs.developer.tech.gov.sg/docs/security-suite-for-engineering-endpoint-devices/support/troubleshooting-issues).
```
[the remark this answers is not in the document]
Comment (kpslp) [reply support/troubleshooting-issues.md:21]: you mean the actual one from Cloudflare?

File: support/troubleshooting-issues.md (hunk 3 above)
Comment (kpslp) [reply support/troubleshooting-issues.md:72]: govind, from which step to which?

File: images/cf-running.png
[the remark this answers is not in the document]
Comment (kpslp) [reply images/cf-running.png:1]: done

File: support/troubleshooting-issues.md (hunk 5 above)
[the remark this answers is not in the document]
Comment (cricketer94) [reply support/troubleshooting-issues.md:21]: Hmmm I am thinking how to reference the other Cloudflare related documentation below. That was what I wanted to make reference to here.

File: support/troubleshooting-issues.md (hunk 3 above)
Comment (cricketer94) [reply support/troubleshooting-issues.md:72]: Ah, so the steps are as above. I just wanted to copy the code block from Phase A step 3 and paste it for the step 3 here.

**Review by cricketer94 — APPROVED** [review]
LGTM. Approved! Thank you for the quick work on this 🙏

**Pull request merged**